projects/github-actions/repo-gardening/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.3] - 2021-05-28
+### Fixed
+- Slack notification tasks: both tasks now listen for pull_request_target events so they can be run on PRs open from forks.
```

Comment:
I wonder if this should be more than a patch bump, since it requires users to start listening for `labeled` events on `pull_request_target` to keep the tasks working.